### ISSUE TITLE
1.1.0 regression bug

### ISSUE THREAD

**Issue by gunta** [rubenv/angular-gettext#107]
1.0.1 works
1.1.0 breaks
latest commit breaks

Example:
```html
<span translate>Some text</span>
```

I can reproduce it in some parts of my app.

What I get in 1.0.1 (correct):
```
Einige Text
```

What I get in 1.1.0 (bug):
```
Einige TextSome text
```

So basically the original text doesn't get removed from the element.

**Comment by carlosgalindo** [comment]
Please see my comment @ https://github.com/rubenv/angular-gettext/issues/161#issuecomment-87016083

**Comment by rubenv** [comment]
This should be fixed by `v2.1.0`. Please test it. If it doesn't work: let us know.
